app/modules/screen-sharing/controllers/CallpreviewCtrl.js

```diff
@@ -0,0 +1,79 @@
+App.controllers.callPreviewCtrl = (function ($, App) {
```

Comment:
Do you still need this ? Surely we're not previewing the screenshare? Or are we?